bundles/binding/org.openhab.binding.ekey/README.md

```diff
@@ -1,97 +1,140 @@
 # ekey Binding
 
-[ekey](http://ekey.net/) is an Austrian company that provides biometric access-control solutions, more precisely fingerprint readers and corresponding controllers.  This binding extends the functionality of the products [ekey home](http://ekey.net/ekey-home-en) and [ekey multi](http://ekey.net/ekey-multi-en).
+## Introduction
```

Comment:
There should never be a header directly underneath the "binding" header - a first introductory paragraph is expected instead (just as it was before), so please remove this line again.

Reply:
Done.